[3.2] docs(textarea): Revamped TextArea Overview (#829)
telerik/blazor-docs: components/textarea/overview.md

> The Telerik <a href ="https://www.telerik.com/blazor-ui/textarea" target="_blank">Blazor TextArea component</a> is a highly customizable multi-line text input area. It provides features like auto resizing based on the user input, floating label functionality and [events]({%slug textarea-events%}) to respond to user actions.

Reviewer: ```suggestion
The Telerik <a href ="https://www.telerik.com/blazor-ui/textarea" target="_blank">Blazor TextArea component</a> is a highly customizable multi-line text input area. It provides features like auto resizing based on the user input and [events]({%slug textarea-events%}) to respond to user actions.
```

## Creating Blazor TextArea

1. Add the `<TelerikTextArea>` tag to a Razor file.

2. Set the `Value` parameter to the desired `string`. It supports one-way and two-way binding.

> 3. (optional) Set the `Label` and `AutoSize` properties to adjust the TextArea height based on the user input.

Reviewer: Label ??

Author: It's forgotten there..

}
````

## Appearance

The TextArea component provides settings to control its appearance. [Read more about the Blazor TextArea appearance settings]({%slug TextArea-appearance%}).

## Events

> The Blazor TextArea fires blur and value change events for further customizing its behavior. [Read more about the Blazor TextArea events]({%slug textarea-events%}).

Reviewer: ```suggestion
The Blazor TextArea fires **blur** and value **change** events to respond to user actions. [Read more about the Blazor TextArea events]({%slug textarea-events%}).
```

| `Id` | `string` | Renders as the `id` attribute on the `<textarea>` element, so you can attach a `<label for="">`. |
| `Label` | `string` | The `label` element rendered next to the `<textarea>` to provide the user with information on its purpose. It covers the input in a fashion similar to a placeholder, and floating animation upwards on focus. If you don't want this effect or the height increase it causes, use the `Id` parameter to attach your own `<label>` tag. |
| `Name` | `string` | The `name` attribute of the HTML element. It is usually required so the `AutoComplete` will be honored by the browser. |
| `PlaceHolder` | `string` | A `string` that maps to the `placeholder` attribute of the HTML element. If a `Label` is defined, it will be shown instead of the placeholder when the input is not focused. |
| `TabIndex` | `Nullable<int>` | Maps to the `tabindex` attribute of the HTML element. You can use it to customize the order in which the inputs in your form focus with the `Tab` key. |
| `Title` | `string` | Maps to the `title` attribute of the HTML element. You can use it to add a [tooltip]({%slug tooltip-overview%}). |

> | `ValidateOn` | `ValidationEvent` | Configures the event that will trigger validation (if validation is enabled). Read more at [Validation Modes for Simple Inputs]({%slug common-features/input-validation%}#validation-modes-for-simple-inputs). |

Reviewer: default value
